src/test/java/org/example/controller/JobRoleControllerTest.java

```diff
@@ -0,0 +1,80 @@
+package org.example.controller;
+
+import org.example.controllers.JobRoleController;
+import org.example.exceptions.DatabaseConnectionException;
+import org.example.models.Band;
+import org.example.models.Capability;
+import org.example.models.OpenJobRoleResponse;
+import org.example.models.Locations;
+import org.example.services.JobRoleService;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import javax.ws.rs.core.Response;
+import java.sql.Date;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class JobRoleControllerTest {
+    JobRoleService jobRoleService = Mockito.mock(JobRoleService.class);
+
+    private final JobRoleController jobRoleController =
+            new JobRoleController(jobRoleService);
+
+    private final OpenJobRoleResponse jobRoleResponse = new OpenJobRoleResponse(
+            "MaryJane1",
+            Locations.BELFAST,
+            Capability.DELIVERY,
+            Band.BAND2,
+            new Date(2024 - 7 - 15));
+
+    /*
+      When the service getOpenJobRoles returns a JobRoleResponse List
+      Expect JobRoleResponse List to be returned in response to GetJobRoles
+     */
+    @Test
+    void getJobRoles_shouldReturnJobRolesResponseList_whenGetAllJobRolesReturnsJobRoleResponseList()
+            throws
+            SQLException, DatabaseConnectionException {
+        List<OpenJobRoleResponse> jobRoleResponseList = new ArrayList<>();
+
+        when(jobRoleService.getOpenJobRoles()).thenReturn(jobRoleResponseList);
+
+        Response response = jobRoleController.getJobRoles();
+
+        assertEquals(jobRoleResponseList, response.getEntity());
```

Comment:
check 200 status code as well

Reply:
added additional assert statement to check 200 status